libs/media-player/src/lib/MediaPlayer.module.scss

```diff
@@ -21,3 +21,8 @@
     color: red;
     font-size: 28px !important;
 }
+
+.media-controls {
```

Comment:
Geoff was wondering if this module is configured and working? It's relevant to the upcoming design decision.

Reply:
Yes, first I imported the stylesheet as

 "import styles from './MediaPlayer.module.scss';"

then the className in the tsx is className={styles['media-controls']}